src/Pelagos/Bundle/AppBundle/Resources/public/js/data-discovery.js

```diff
@@ -99,8 +106,213 @@ function resizeLeftRight() {
     $("#right").height(rh);
 }
 
+function formatBytes(bytes,decimals) {
+    if(bytes == 0) return '0 Bytes';
+    var k = 1024,
+        dm = decimals <= 0 ? 0 : decimals || 2,
+        sizes = ['Bytes', 'KB', 'MB', 'GB', 'TB', 'PB', 'EB', 'ZB', 'YB'],
```

Comment:
I think for consistency we decided to use double quotes in Javascript

Reply:
yup copy/paste code